components/BottomNavBar.js

```diff
@@ -0,0 +1,42 @@
+import { View, Text, StyleSheet } from 'react-native';
+
+
+const BottomNavBar = () => {
+
+    return (
+        <View style={styles.bottomNavBar}>
+            <Text style={styles.title}> 🏠      📅      🪷      📄 </Text>
+        </View>
+    )
+}
+
+export default BottomNavBar;
+
+const styles = StyleSheet.create({
+    bottomNavBar: {
+        backgroundColor: '#39376E',
+        // marginBottom: 380,
+        height: 80,
+        width: 400,
+        // alignItems: 'center',
+        justifyContent: 'center',
+        borderWidth: 1,
+        borderColor: '#39376E',
+        borderRadius: 18,
+        shadowColor: "#869684",
+        shadowOffset: {
+        width: 2,
+        height: 4,
+        },
+        shadowOpacity: .8,
+        shadowRadius: 5,
+        elevation: 10,
+    },
+
+    title: {
+        // fontFamily: ??? needs to be imported
```

Comment:
Font family still needs to be worked in... also I left docs in the Slack channel for the background image. I know we'll figure it out. Just thinking ahead!